### ISSUE TITLE
sirius search error

### ISSUE THREAD

**Issue by negarg** [mzmine/mzmine2#662]
### Setup

I am reporting a problem with MZmine 2 version 2.40.1
My operating system is windows 10 version 1809 64-bit 

I am clicking on single feature and then sirius structure prediction. I get the attached error
![sirius search error](https://user-images.githubusercontent.com/17147250/61973914-47325b00-afb3-11e9-9e35-82e6b0976339.PNG)

Below are the parameters used
![parameters for search](https://user-images.githubusercontent.com/17147250/61973980-70eb8200-afb3-11e9-8f30-28d6cca21452.PNG)

**Comment by tomas-pluskal** [comment]
Seems like same issue as #526

**Comment by tomas-pluskal** [comment]
Could you try moving your MZmine installation to C:\MZmine2 and starting from there?

**Comment by tomas-pluskal** [comment]
Also please try using MZmine 2.41 and upload the complete log file after running MZmine (found in the log folder under MZmine folder).

**Comment by negarg** [comment]
So, I moved it to c\MZmine2
did not help. Log file is attached.
[mzmine.log](https://github.com/mzmine/mzmine2/files/3437098/mzmine.log)


I installed version 2.41 and it gives errors while using a batch file that worked well with version 2.40. SO, I am guessing batch files between versions
![sirius version 2 41 errors with batch file](https://user-images.githubusercontent.com/17147250/61975564-7ea30680-afb7-11e9-828d-286e6c43f8ca.png)
 are not compatible
Errors being:

**Comment by tomas-pluskal** [comment]
If new parameters have been added to some modules, you might need to update those batch steps to reflect the new parameters.

**Comment by tomas-pluskal** [comment]
In the log file you uploaded I don't see any error. It seem you used the SIRIUS export module instead of the SIRIUS search module.

**Comment by negarg** [comment]
Hi Tomas. 

Thank you! I did not use the sirius export module.. I used the search one. The screenshot below shows it is the search module
![parameters for search](https://user-images.githubusercontent.com/17147250/61977717-37b80f80-afbd-11e9-9448-0053f47a5734.PNG)

As to why the error is not in log file. I am not sure. But the error below pops up on the screen.
![sirius search error](https://user-images.githubusercontent.com/17147250/61977763-55857480-afbd-11e9-82d6-bbff1aa2dcbd.PNG)
Latest log file
[mzmine.log](https://github.com/mzmine/mzmine2/files/3437230/mzmine.log)

I will try fixing the steps of batch file and evaluate if new version of mzmine fixes this problem
